src/pages/generative/[id]/collection.tsx

```diff
@@ -69,7 +69,7 @@ const GenerativeTokenCollection: NextPage<Props> = ({ token }) => {
               <Spacing size="large"/>
               <Link href={getGenerativeTokenUrl(token)}>
                 <Button isLink={true} size="small">
-                  See Generative Token
+                  See collection
```

Comment:
I find this update unclear.
The collection of a Generative Token is already a page (/generative/:id/collection)
This link points to the Generative Token itself, not the collection